openedx/core/djangoapps/schedules/management/commands/tests/test_send_email_base_command.py

```diff
@@ -33,6 +34,7 @@ def test_handle(self):
             send_emails.assert_called_once_with(
                 self.site,
                 datetime.datetime(2017, 9, 29, tzinfo=pytz.UTC),
+                None,
                 None
```

Comment:
Changed it to False.

Reply:
Actually this was because there were failing test case, where some calls were made without the `check_completion` variable.

Reply:
That feels odd to me. Because in the resolver, you define the default value as `False`, not None. Is there a chance there is another spot where it is defaulting to None and that should be updated to False? I'd love to have consistency here.